testing/test_cli_sdk.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+"""
+Exercise selected functions from cli_sdk to improve coverage
+"""
+
+import cli_sdk
+
+import pytest
+
+intent = "digit"
+transcript = "please dial eight"
+expected_response = {
+    "transcript": transcript,
+    "intents": [
+        {
+            "label": intent,
+            "probability": 1.0,
+            "entities": [
+                {
+                    "label": "room_number",
+                    "matches": [
+                        {
+                            "value": "8",
+                            "probability": 1.0,
+                            "lattice_path": [[2, 0, None]],
+                            "startTimeSec": 2.0,
+                            "endTimeSec": 3.0,
+                            "interpreted_transcript": "eight",
+                        }
+                    ],
+                }
+            ],
+        }
+    ],
+}
+
+class TestCLISDK:
+    def test_get_response(self):
+        "Ensures responses behave as expected"
+
+        assert cli_sdk.get_response(intent, transcript, raw_data=False) is None
+        assert (
+            cli_sdk.get_response(intent, transcript, raw_data=True) == expected_response
+        )
+
+    def test_get_entities(self):
+        "Ensures we can fetch entities"
+        # get_entities
+        cli_sdk.get_entities(intent, transcript)
+        resp = cli_sdk.get_entities(intent, transcript, raw_data=True)
+        assert resp.columns.tolist() == ["entity", "value", "tokens", "indices"]
+        assert resp.values.tolist() == [["room_number", "8", "eight", [2]]]
+
+    def test_restart(self):
+        "Ensures restarts and reloads work"
+        cli_sdk.restart("examples/digit/custom")
+        cli_sdk.reload()
+        resp = cli_sdk.get_response(intent, transcript, raw_data=True)
+        assert resp == expected_response
+
+
+    def test_log(self):
+        "Tests log fetching"
+        assert cli_sdk.log() != ""
```

Comment:
Do we need to make sure this isn't `None` or something? maybe make sure it's a string?

Reply:
Is checking type and length>0 sufficient?